### PR TITLE
aircrack-ng: add Chinese translation

### PR DESCRIPTION
**Pull request by tianyeeT**
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages.zh/common/aircrack-ng.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# aircrack-ng
+
+> 破解捕获数据包中的握手时段的 WEP 和 WPA/WPA2 密钥。
+> 是 Aircrack-ng 网络软件套件的一部分。
+> 更多信息： <https://www.aircrack-ng.org/doku.php?id=aircrack-ng>。
```
Comment (blueskyson) [inline pages.zh/common/aircrack-ng.md:5]:
```suggestion
> 更多信息： <https://www.aircrack-ng.org/doku.php?id=aircrack-ng>.
```

File: pages.zh/common/aircrack-ng.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- 使用字典文件破解捕获文件中的密钥：
+
+`aircrack-ng -w {{path/to/wordlist.txt}} {{path/to/capture.cap}}`
```
Comment (blueskyson) [inline pages.zh/common/aircrack-ng.md:9]:
```suggestion
`aircrack-ng -w {{路径/到/字典文件.txt}} {{路径/到/捕获文件.cap}}`
```

File: pages.zh/common/aircrack-ng.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+- 使用字典文件和接入点的 ESSID 破解捕获文件中的密钥：
+
+`aircrack-ng -w {{path/to/wordlist.txt}} -e {{essid}} {{path/to/capture.cap}}`
```
Comment (blueskyson) [inline pages.zh/common/aircrack-ng.md:13]:
```suggestion
`aircrack-ng -w {{路径/到/字典文件.txt}} -e {{essid}} {{路径/到/捕获文件.cap}}`
```

File: pages.zh/common/aircrack-ng.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+- 使用字典文件和接入点的 MAC 地址破解捕获文件中的密钥：
+
+`aircrack-ng -w {{path/to/wordlist.txt}} --bssid {{mac}} {{path/to/capture.cap}}`
```
Comment (blueskyson) [inline pages.zh/common/aircrack-ng.md:17]:
```suggestion
`aircrack-ng -w {{路径/到/字典文件.txt}} --bssid {{mac}} {{路径/到/捕获文件.cap}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by spageektti — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**